ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -1773,6 +1773,15 @@ management_console:
   idle_timeout: < 0-86400 in minutes >
 ```
 
+#### Management CVX
+```yaml
+management_cvx:
+  shutdown: false
+  server_host:
```

Comment:
Make this knob **server_hosts** because it is a list of hosts. Also add a comment above this (line1780) to explain the context of this knob.

My suggestion: ```# List of Hostname(s) or IP Address(es) of CVX server(s)```